gitlab/resource_gitlab_group_label.go

```diff
@@ -123,3 +129,25 @@ func resourceGitlabGroupLabelDelete(d *schema.ResourceData, meta interface{}) er
 	_, err := client.GroupLabels.DeleteGroupLabel(group, options)
 	return err
 }
+
+func resourceGitlabGroupLabelImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	client := meta.(*gitlab.Client)
+	parts := strings.SplitN(d.Id(), ":", 2)
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid label id (should be <group ID>.<label name>): %s", d.Id())
+	}
+
+	d.SetId(parts[1])
+	group, _, err := client.Groups.GetGroup(parts[0])
+	if err != nil {
+		return nil, err
+	}
+
+	if err := d.Set("group", strconv.Itoa(group.ID)); err != nil {
+		return nil, err
+	}
+
+	err = resourceGitlabGroupLabelRead(d, meta)
```

Comment:
Is it necessary to read here? Doesn't Terraform perform a read for you before the next operation anyway? I don't think it changes the provider behavior.

Reply:
Never mind. I see it is a recommended practice in Terraform docs. Carry on!

Reply:
Possibly... I was modelling this on how all the other Importers were implemented?

Reply:
Yes I think it makes sense to read and make sure the resource you try to import exists.